### ISSUE TITLE
Setup PythonPath explicitly in test

### ISSUE THREAD

**Issue by francoisjamet** [TRIQS/triqs#499]
Dear experts,

I'm trying to install the unstable branch on my university's cluster.
I've already installed the stable branch and it compiled well.

During the compilation of the unstable branch, I get : 
```
[k1634343@henrya10(gravity) triqs_u]$ make -j32
Scanning dependencies of target py_copy
Scanning dependencies of target py_copy_test
Scanning dependencies of target gtest
Scanning dependencies of target triqs
[  1%] Generating ../py_copy_test.tar
.....

[ 93%] Built target scope_exit
[ 93%] Built target operator_test
[ 93%] Built target variant_int_string
[ 93%] Linking CXX executable ser
[ 93%] Linking CXX executable p_gg_new
[ 93%] Built target ser
[ 93%] Linking CXX executable h5_seria
[ 93%] Built target p_gg_new
[ 93%] Built target h5_seria
[ 93%] Linking CXX executable variant
[ 93%] Built target variant
[ 93%] Linking CXX shared module my_module.so
[ 93%] Linking CXX executable bubble
[ 93%] Built target my_module
[ 94%] Generating my_moduleB_wrap.cxx
[ 94%] Built target bubble
[ 95%] Linking CXX shared module extractors.so
Traceback (most recent call last):
  File "/users/k1634343/src/triqs.src/test/pytriqs/wrap_test/my_moduleB_desc.py", line 3, in <module>
    import my_module
ImportError: No module named my_module
make[2]: *** [test/pytriqs/wrap_test/my_moduleB_wrap.cxx] Error 1
make[1]: *** [test/pytriqs/wrap_test/CMakeFiles/my_moduleB.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 95%] Built target extractors
[ 95%] Linking CXX shared module meshes.so
[ 95%] Built target meshes
[ 96%] Linking CXX shared module singularities.so
[ 96%] Built target singularities
make: *** [all] Error 2
```

on my laptop (archlinux), this branch compiles well so it comes from the cluster configuration.

the cmake output is :
```1634343@henrya10(gravity) triqs_u]$ env CC=gcc GXX=g++ LDFLAGS="-lmkl_rt -lmkl_gf_lp64 -lmkl_gnu_thread -lmkl_core -liomp5 -lifcore -lpthread" BOOST_ROOT=/users/k1222090/opt/boost/1.57.0/impi/5.0.3.049/intel/15.0.4/o cmake -DCMAKE_INSTALL_PREFIX=~/opt/triqs_u $HOME/src/triqs.src
-- Installation directory will be ~/opt/triqs_u
-- -------- Checking compiler C++ status-------------
-- Build type: Release
-- -------- Cpp2Py detection -------------
-- Found Cpp2pyConfig.cmake with version 1.5, hash = cb939d260272a6855adc8dfc116c90ffac495701
-- -------- git and hash detection -------------
-- Found Git: /usr/bin/git (found version "1.7.1") 
-- Git hash : 7cc19fb3e94559e929bfb96d75cfa8fd33ef14ae
-- Hostname : henrya10.prv.gravity.cluster
-- Compiled by : k1634343
-- -------- MPI detection -------------
-- Found MPI_C: /users/k1222090/opt/openmpi/2.1.1/gcc/6.1.0/o/lib/libmpi.so  
-- Found MPI_CXX: /users/k1222090/opt/openmpi/2.1.1/gcc/6.1.0/o/lib/libmpi.so  
-- MPI C compiler : /users/k1222090/opt/openmpi/2.1.1/gcc/6.1.0/o/bin/mpicc
-- MPI_COMPILE_FLAGS :  
-- MPI_C_INCLUDE_PATH : /users/k1222090/opt/openmpi/2.1.1/gcc/6.1.0/o/include
-- MPI_C_LIBRARIES : /users/k1222090/opt/openmpi/2.1.1/gcc/6.1.0/o/lib/libmpi.so
-- MPI_CXX_LIBRARIES : /users/k1222090/opt/openmpi/2.1.1/gcc/6.1.0/o/lib/libmpi.so
-- -------- Boost detection -------------
-- Boost version: 1.57.0
-- Boost include dir: /users/k1222090/opt/boost/1.57.0/impi/5.0.3.049/intel/15.0.4/o/include
-- Boost libraries: 
-- -------- Misc -------------
-- Checked max_align_t. No workaround needed
-- Using <experimental/optional> header instead of <optional>
-- -------- Lapack detection -------------
-- The Fortran compiler identification is GNU 6.1.0
-- Check for working Fortran compiler: /users/k1222090/opt/gcc/6.1.0/bin/gfortran
-- Check for working Fortran compiler: /users/k1222090/opt/gcc/6.1.0/bin/gfortran  -- works
-- Detecting Fortran compiler ABI info
-- Detecting Fortran compiler ABI info - done
-- Checking whether /users/k1222090/opt/gcc/6.1.0/bin/gfortran supports Fortran 90
-- Checking whether /users/k1222090/opt/gcc/6.1.0/bin/gfortran supports Fortran 90 -- yes
-- Falling back to CMake provied LAPACK/BLAS detection.
-- Looking for Fortran sgemm
-- Looking for Fortran sgemm - found
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - found
-- Found Threads: TRUE  
-- A library with BLAS API found.
-- A library with BLAS API found.
-- Looking for Fortran cheev
-- Looking for Fortran cheev - found
-- A library with LAPACK API found.
-- Lapack libraries : /usr/lib64/liblapack.so;/usr/lib64/libblas.so;/usr/lib64/libpthread.so;/usr/lib64/libblas.so
-- -------- HDF5 detection -------------
-- Found HDF5: /users/k1222090/opt/hdf5/1.8.18/gcc/6.1.0/o/lib/libhdf5.so (found version "1.8.18") found components:  C HL 
--  HDF5_LIBRARIES = /users/k1222090/opt/hdf5/1.8.18/gcc/6.1.0/o/lib/libhdf5.so;/users/k1222090/opt/hdf5/1.8.18/gcc/6.1.0/o/lib/libhdf5_hl.so
-- hdf5 version: 1.8.18
-- -------- GMP detection -------------
-- GMP libs: /usr/lib64/libgmp.so /usr/lib64/libgmpxx.so
-- Found GMP: /users/k1222090/opt/gcc/6.1.0/include  
-- -------- FFTW detection -------------
-- Found FFTW: /users/k1222090/opt/fftw3/3.3.6/gcc/6.1.0/o/lib/libfftw3.so  
-- -------- Preparing python extension modules  -------------
-- Adding cpp2py Python module block_matrix
-- Adding cpp2py Python module meshes
-- Adding cpp2py Python module singularities
-- Adding cpp2py Python module gf_fnt
-- Adding cpp2py Python module wrapped_aux
-- Adding cpp2py Python module lattice_tools
-- Adding cpp2py Python module operators
-- Adding cpp2py Python module extractors
-- Adding cpp2py Python module random_generator
-- Adding cpp2py Python module histograms
-- -------- Preparing tests  -------------
-- Adding cpp2py Python module my_module
-- Adding cpp2py Python module my_moduleB
-- Adding cpp2py Python module test_g
-- Adding cpp2py Python module test_bl
-- Adding cpp2py Python module test_multivar
-- -------- Making triqs_config.h  -------------
-- -------- Making TRIQSConfig.cmake -------------
-- ***************************************************************
-- * Use :                                                        
-- *   source /users/k1634343/opt/triqs_u/share/triqsvars.sh          
-- *                                                              
-- *   to set up the environment variables                        
-- ***************************************************************
-- Configuring done
-- Generating done
-- Build files have been written to: /users/k1634343/dev/triqs_u
```

Thanks in advance for your help,

Francois

**Comment by parcollet** [comment]
Hi, 
Most likely it is due to the fact that on this machine ./ is not in the  PYTHONPATH.
We can not reproduce the issue on e.g. our Travis CI.

Try some : 
export PYTHONPATH=./:$PYTHONPATH

Renaming  the issue.
--> port the wrap test to cthyb and use the correct Cmakelist from ctint/cthyb
to set up the path by default.

**Comment by francoisjamet** [comment]
Dear Olivier,

it works perfectly!
thanks for your help and for this amazing tool !
Francois

**Comment by parcollet** [comment]
Thanks.
Keeping it open to add a little fix in calling the test to 
avoid the pb in the future.

**Comment by Wentzell** [comment]
Fixed by abee0a8410e3c9c5a4091ece515afd5a45e471bb
